src/screens/Loading/index.js

```diff
@@ -1,31 +1,31 @@
 import * as React from 'react';
 import {ActivityIndicator, Modal, StyleSheet, View} from 'react-native';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {COLORS} from '../../utils/theme';
 
 const Loading = ({visible}) => {
   return (
     <Modal transparent visible={visible}>
       <View style={styles.container}>
         <View style={styles.content}>
-          <ActivityIndicator size="large" color={COLORS.holyTosca} />
+          <ActivityIndicator size="large" color={COLORS.holytosca} />
         </View>
       </View>
     </Modal>
   );
 };
 
-export default withInteractionsManaged (React.memo (Loading));
+export default withInteractionsManaged(React.memo(Loading));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The export statement has been modified. The spaces around the exported component have been removed, which is a good practice as it improves readability and consistency in the code.
```diff
- export default withInteractionsManaged (React.memo (Loading));
+ export default withInteractionsManaged(React.memo(Loading));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->